app/views/coronavirus_form/expert_advice.html.erb

```diff
@@ -0,0 +1,67 @@
+<% content_for :title do %><%= t('coronavirus_form.expert_advice.title') %><% end %>
+    <% content_for :meta_tags do %>
+      <meta name="description" content="<%= t('coronavirus_form.expert_advice.title') %>" />
+    <% end %>
+    
+    <% content_for :back_link do %>
+      <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+    <% end %>
+    
+    <%= render "coronavirus_form/validation_error" %>
```

Comment:
This should be removed as we now have it in the layout, otherwise we'll get the error messages twice.

Reply:
Ah sorry Alex, I'll fix that in another PR.

Reply:
Added here https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/pull/37